.github/workflows/terraform-ci-deploy.yml

```diff
@@ -82,9 +82,9 @@ jobs:
     if: needs.paths-filter.outputs.operations == 'true' && always()
     runs-on: ubuntu-latest
     env:
-      ARM_CLIENT_ID: ${{ secrets.AZURE_CLIENT_ID }}
-      ARM_TENANT_ID: ${{ secrets.AZURE_TENANT_ID }}
-      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+      ARM_CLIENT_ID: ${{ secrets.AZURE_CDC_CLIENT_ID }}
+      ARM_TENANT_ID: ${{ secrets.AZURE_CDC_TENANT_ID }}
+      ARM_SUBSCRIPTION_ID: ${{ secrets.AZURE_CDC_DMZ_C1_SUBSCRIPTION_ID }}
```

Comment:
Same here, needs to be reverted because the PR environment is in the Flexion Entra domain.